src/common/Input/Input.js

```diff
@@ -93,6 +96,10 @@ const Input = React.forwardRef(
         setInputIsFocused(false)
 
         onBlur(event)
+
+        if (required && !typedValue) {
```

Comment:
please rename function to inputOnBlur

```suggestion
        if (required && typedValue.length === 0) {
```